manifests/m/matchmycolor/Colibri/23.2.1.12175/matchmycolor.Colibri.locale.en-US.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: matchmycolor.Colibri
+PackageVersion: 23.2.1.12175
+PackageLocale: en-US
+Publisher: matchmycolor

```

Comment:
```suggestion
Publisher: matchmycolor LLC
```

Reply:
The first package we created had matchmycolor LLC for the publisher. We then changed it to matchmycolor. E.g. our latest package (23.2.0.12171) also has matchmycolor for the publisher. Can we keep matchmycolor for publisher?